html/Axit/index.html

```diff
@@ -0,0 +1,591 @@
+            <li class="price__option">email support</li>
+          </ul>
+        </button>
+       
+        <!-- Price 2 -->
+        <button class="price pricing__price">
+         
+          <p class="price__title">Professional</p>
+
+          <div class="price__money">
+            
+            <p class="price__number">
+              $ <span>19</span>
+            </p>
+            
+            <p class="description  price__period">Monthly Payment</p>
+
+            <p class="price__popularity price__popularity--with-text">
+              OUR MOST POPULAR
+            </p>
+          </div>
+
+          <ul class="price__options">
+            <li class="price__option">5 gb of space</li>
+            <li class="price__option">50 gb of bandwidth</li>
+            <li class="price__option">12 websites</li>
+            <li class="price__option">Advanced customization</li>
+            <li class="price__option">wordpress intergration</li>
+            <li class="price__option">email support</li>
+          </ul>
+         
+        </button>
+       
+        <!-- Price 3 -->
+        <button class="price pricing__price">
+          <p class="price__title">Enterprice</p>
+  
+          <div class="price__money">
+            
+            <p class="price__number">
+              $ <span>7</span><span>0</span>
+            </p>
+            
+            <p class="description  price__period">Monthly Payment</p>
+
+            <p class="price__popularity">
+            </p>
+          </div>
+
+          <ul class="price__options">
+            <li class="price__option">unlimited space</li>
+            <li class="price__option">unlimited bandwidth</li>
+            <li class="price__option">100 websites</li>
+            <li class="price__option">advanced customization</li>
+            <li class="price__option">wordpress intergration</li>
+            <li class="price__option">24/7 customer support</li>
+          </ul>
+        </button >
+
+      </div>
+  </section>
+
+  <!-- Customers -->
+  <section class="customers">
+    <h2 class="customers__title title title--big">
+      WHAT OUR CUSTOMERS ARE SAYING
+    </h2>
+
+    <span class="separator customers__separator"></span>
+
+    <p class="description customers__description">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    </p>
+
+    <div class="customers__container">
+
+      <!-- Customer 1 -->
+      <div class="customers__customer customer">
+        <p class="description customer__opinion">
+          Lorem ipsum dolor sit amet, 
+          consectetur adipisicing elit. Doloribus 
+          accusamus expedita repellat similique 
+          odio aspernatur ex, architecto eaque 
+          quo suscipit.
+          <span class="customer__indicator"></span>
+        </p>
+        <div class="customer__details">
+          <img src="./assets/customer/avatar-2-u-231@3x.png" alt="Jermey H.">
+          <div class="customer__names">
```

Comment:
not sure if customer position is part of his name, maybe some more general class name would be better here